backend/routes/supportRoutes.js

```
@@ -15,6 +15,7 @@ const {
   getUserMemberships
 } = require('../controllers/supportController');
 const { protect } = require('../middleware/authMiddleware');
+const { requireRole } = require('../middleware/roleMiddleware');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'requireRole'. Consider to delete it or rename it.

'requireRole' is defined here:


(lint/suspicious/noRedeclare)